certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1283 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Alkymi",
+    "description": "Alkymi transforms enterprise workflows by placing machine learning and automation technology into the hands of business \nusers, turning operational data headaches into a source of efficiency. Alkymi’s core products—Data Inbox and Patterns—\nallow businesses to extract, analyze, and take action on critical business data in emails and documents.\n\nThe Alkymi connector allows you to easily access the data extracted from unstructured documents processed by the Alkymi \napp. This connector will allow you to easily pipe data from the external API to any of the apps in the Microsoft \nPowerApps ecosystem.",
```

Comment:
@tori-alkymi Please remove html tags of newline (\n) from description values, they are just placed randomly in the description.